packages/web-components/src/components/cbp-list/cbp-list.scss

```diff
@@ -0,0 +1,68 @@
+/**
+ * @prop --cbp-list-item-color: var(--cbp-color-text-darkest);
+ * @prop --cbp-list-item-color-dark: var(--cbp-color-text-lightest);
+*/
+
+:root {
+  --cbp-list-item-color: var(--cbp-color-text-darkest);
+  --cbp-list-item-color-dark: var(--cbp-color-text-lightest);
+  --cbp-list-padding: var(--cbp-space-4x);
+  --cbp-list-item-padding: 2px;
+  // --cbp-list-item-bullet: '-';
+}
+
+
+[data-cbp-theme=light] cbp-list[context*=dark]:not([context=light-always]),
+[data-cbp-theme=dark] cbp-list:not([context=dark-inverts]):not([context=light-always]) {
+  --cbp-list-item-color: var(--cbp-list-item-color-dark);
+}
+
+cbp-list {
+  line-height: var(--cbp-line-height-xs);
+
+  ol, ul{
+    padding-inline-start: var(--cbp-list-padding);
+  }
+  
+  li {
+    color: var(--cbp-list-item-color);
+    font-size: var(--cbp-font-size-body);
+    font-weight: var(--cbp-font-weight-bold);
+    padding-block: var(--cbp-list-item-padding);
+  }
+
+  
+  // //Unordered nested lists
+  ul > li{
+    list-style-type: disc;
+   
+    ul li{ //Unordered list 1st sublist
+      list-style-type: '- ';
+    }
+  }
+    
+  //Ordered nested lists
+  ol > li{
+    list-style-type: decimal;
+
+    ol li{ //Ordered list 1st sublist
+      list-style-type: upper-alpha; 
+    
+      ol li{ //Ordered list 2nd sublist
+        list-style-type: decimal; 
+    
+        ol{
+          counter-reset: count;
+    
+          li{//Ordered list 3rd sublist
+            counter-increment: count;
+             
+            &::marker{ 
```

Comment:
This one is too deep. It's currently on 4th-level, but should be on 3rd. 4th is small alpha.